doc/userdoc/guides/random_numbers.rst

```diff
@@ -45,7 +45,7 @@ How to work with randomness in NEST
 NEST makes randomness easy: Setting a single seed gives you full
 control over randomness throughout a simulation, and you can choose seed values
 at liberty. NEST's powerful mechanism to :ref:`parameterize nodes and synapses <param_ex>`
-and NEST's :ref:`probabilistic connection rules <connection_mgnt>` allow you
+and NEST's :ref:`probabilistic connection rules <connection_management>` allow you
```

Comment:
Won't be better to make the name and the reference have similar meanings.
```suggestion
and NEST's :ref:`probabilistic connection properties <network_with_random_properties>` allow you
```

Reply:
```suggestion
and NEST's :ref:`probabilistic connection rules <conn_rules>` allow you
```

Reply:
The label has to reference an existing reference. I'd suggest specifying the section on connection rules in the connection management guide.